### PR TITLE
Add calendar for landingview

### PR DESCRIPTION
**Pull request by Frlundsten**
Display what movies are scheduled for that day

jira PG2-10

### CODE REVIEW

File: frontend/src/views/LandingView.vue [hunk 1]
```diff
@@ -77,13 +78,27 @@ function displaySearch() {
...
 
 <template>
   <main class="p-0 h-full relative">
     <div class="min-h-screen h-full w-full bg-[length:600%] md:bg-[length:300%] lg:bg-[length:100%] z-0 bg-[bottom_6rem_right] lg:bg-[bottom_right] bg-fixed bg-[url('/img/cinemabg.jpg')]">
       <!-- Length is needed to properly "zoom in" on the image -->
       <!-- This div only contains the background image -->
+       <div  class="text-center">
+        <div class="cursor-pointer" @click="showMoviePlanner()">
+        <h4 v-if="!isShowingPlanner">Show calendar</h4>
+         <span :style="{ transform: isShowingPlanner ? 'rotate(90deg)' : 'rotate(-90deg)' }" class="rotate-90 text-6xl material-symbols-outlined">
```
Comment (Emmelie83) [inline frontend/src/views/LandingView.vue]: Suggesting a smaller size (text-4xl)

File: frontend/src/views/LandingView.vue [hunk 2]
```diff
@@ -77,13 +78,27 @@ function displaySearch() {
...
+}
 </script>
 
 <template>
   <main class="p-0 h-full relative">
     <div class="min-h-screen h-full w-full bg-[length:600%] md:bg-[length:300%] lg:bg-[length:100%] z-0 bg-[bottom_6rem_right] lg:bg-[bottom_right] bg-fixed bg-[url('/img/cinemabg.jpg')]">
       <!-- Length is needed to properly "zoom in" on the image -->
       <!-- This div only contains the background image -->
+       <div  class="text-center">
+        <div class="cursor-pointer" @click="showMoviePlanner()">
```
Comment (Emmelie83) [inline frontend/src/views/LandingView.vue]: Maybe some padding to the top (like pt-5)?

**Comment by Emmelie83** [comment]
Nice job! But I think I would prefer to have the calendar after the search field like this:
Screenshot 2024-10-31 184458
What do you think?

**Review by DevelopedByDanny — APPROVED** [review]
Good job

**Review by Mattan41 — APPROVED** [review]
Nicely done!

**Pull request merged**